src/style.css

```diff
@@ -0,0 +1,74 @@
+@font-face {
+  font-family: Avenir;
+  src: url(/fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+body {
+  margin-top: 0;
+}
+
+.big_form {
+  margin-top: 20px;
+  margin-bottom: 20px;
+  position: relative;
+}
+
+.small_form {
+  position: relative;
+}
+
+.big_searchbar {
+  height: 70px;
+  font-size: 16px;
+  padding-left: 62px;
+}
+
+.small_searchbar {
+  height: 42px;
+  font-style: normal;
+  font-size: 14px;
+  padding-left: 33px;
+}
+
+.big_search_icon {
+  position: absolute;
+  width: 19px;
+  top: 25px;
+  left: 26px;
+  background-color: #fff0;
+}
+
+.small_search_icon {
+  width: 11px;
+  position: absolute;
+  top: 15px;
+  left: 13px;
+  background-color: #fff0;
+}
+
+.search {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
```

Comment:
use `*` selector to set `box-sizing` since it should be the same for all elements
```suggestion
```